cmd/brew-test-bot.rb

```diff
@@ -533,12 +533,17 @@ def download
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)", "origin/master"
         ).strip
-        puts "#{@tap} origin/master #{tap_origin_master_revision}"
         tap_revision = Utils.popen_read(
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)"
         ).strip
-        puts "#{@tap} HEAD #{tap_revision}"
+        puts "\nTesting tap #{@tap} with:"
```

Comment:
Use `<<~EOS` which allows you to embed newlines and print multiple lines.

Reply:
Of course! Completely forgot about that :)